### ISSUE TITLE
no killswitch during start and restart

### ISSUE THREAD

**Issue by hermann2971** [jotyGill/openpyn-nordvpn#197]
Thanks for this great tool. I just installed it and did some tests. The killswitch works when no vpn connection is established. But during start or restart there is no blocking and all traffic goes through standard connection. Is there any possibillity to solve this issue?

Thanks a lot!

**Comment by hermann2971** [comment]
Hi,
thank you for your reply. Maybe it is possible to choose if the firewall should be temporary or permanent. And if the firewall is permanent you could ping the url of nordvpn api to get the ip and then create an exception for iptables (maybe ip will not change often). For permanent iptables it would also be great that stop of the service will not flush the tables. At the moment I have to change the service file manually to change behavior - just kill connection but do nut flush.

Thanks a lot

**Comment by ISO-morphism** [comment]
@hermann2971 

> For permanent iptables it would also be great that stop of the service will not flush the tables.

I agree, and I opened an issue for it in #202. I think the ideas of using custom `iptables` chains, and using `iptables -I` to insert rules at the beginning of chains will help in the auto fail-over design.

**Comment by larry77** [comment]
Any progress on this? I see that nordvpn has released a linux app, but I would like to stick to this wonderful open source project!
